pkg/assets/asset.go

```diff
@@ -249,9 +249,9 @@ func GetAssetsJSONPath() (string, error) {
 }
 
 func XMAInfo() models.AssetInfo {
```

Comment:
Shouldn't this function be also renamed ? 

```suggestion
func MASInfo() models.AssetInfo {
```